src-backend/api/tests/utils/decorators.py

```diff
@@ -0,0 +1,9 @@
+import api.startup
+
+# Use this decorator to add permissions to a new user
+# created in your setUp method for tests.
+def add_group_permissions(self):
+    def _run_once():
```

Comment:
There doesn't seem to be a specific convention for decorators: https://github.com/django/django/tree/8047e3666b0b50bb04e6f16c2a4fb21ddfd5713f/django/views/decorators

Some of them use `_method_name` while others use `decorator`